Classes/common/CDTFetchChanges.h

```diff
@@ -0,0 +1,163 @@
+ <dt>revision</dt>
+ <dd>The winning revision for the document that changed.</dd>
+ </dl>
+ 
+ The operation object executes this block once for each document in the database that changed 
+ since the previous fetch request. Each time the block is executed, it is executed 
+ serially with respect to the other progress blocks of the operation. If no documents 
+ changed, the block is not executed.
+ 
+ If you intend to use this block to process results, set it before executing the 
+ operation or submitting it to a queue. 
+ */
+@property (nonatomic, copy) void (^documentChangedBlock)(CDTDocumentRevision *revision);
+
+
+/**
+ The block to execute for each deleted document.
+ 
+ The block returns no value and takes the following parameters:
+ 
+ <dl>
+ <dt>docId</dt>
+ <dd>The document id for the deleted document.</dd>
+ </dl>
+ 
+ The operation object executes this block once for each document in the database that
+ was deleted since the previous fetch request. Each time the block is executed, it 
+ is executed serially with respect to the other progress blocks of the operation. 
+ If no documents were deleted, the block is not executed.
+ 
+ If you intend to use this block to process results, set it before executing the 
+ operation or submitting it to a queue. 
+ */
+@property (nonatomic, copy) void (^documentWithIDWasDeletedBlock)(NSString *docId);
+
+/**
+ The block to execute when all changes have been reported. 
+ 
+ The block returns no value and takes the following parameters:
+ 
+ <dl>
+ <dt>serverSequenceValue</dt>
+ <dd>The new sequence value from the server. You can store this value locally and use it 
+ during subsequent fetch operations to limit the results to records that changed since 
+ this operation executed. A sequence value is only valid for the database it was
+ originally retrieved from.</dd>
+ 
+ <dt>clientSequenceValue</dt>
+ <dd>The sequence value you specified when you initialized the operation object.</dd>
+ 
+ <dt>fetchError</dt>
+ <dd>An error object containing information about a problem, or nil if the changes are 
+ retrieved successfully.</dd>
+ </dl>
+ 
+ The operation object executes this block only once, at the conclusion of the operation. It 
+ is executed after all individual change blocks. 
+ The block is executed serially with respect to the other progress blocks of the operation.
+ 
+ If you intend to use this block to process results, set it before executing the operation or 
+ submitting the operation object to a queue. 
+ */
+@property (nonatomic, copy) void (^fetchRecordChangesCompletionBlock)( 
+NSString *serverSequenceValue, 
+NSString *clientSequenceValue, 
+NSError *fetchError);
+
+#pragma mark Initialisers
+
+/**
+ Initializes and returns an object configured to fetch changes in the specified database.
+ 
+ When initializing the fetch object, use the sequence value from a previous fetch request if 
+ you have one. You can archive sequence values and write them to disk for later use if needed.
+ 
```

Comment:
What's the harm in exposing it? Maybe convert it to an NSString representation? 

Should we at least expose the last sequence? Perhaps ```CDTDatastore -getLastSequence``` would be useful. I am thinking about the app that has a pre-seeded datasbase. Otherwise, the developer would need to run CDTFetchObjects on the app's first launch to scroll through to the end to get the last sequence.